tests/integration/fixtures/mini_sentry.py

```diff
@@ -182,6 +182,12 @@ def add_full_project_config(self, project_id, dsn_public_key=None, extra=None):
         self.project_configs[project_id] = ret_val
         return ret_val
 
+    def set_global_config_option(self, option_name, value):
+        # must be called before initializing relay fixture
+        old_val = self.global_config["options"].get(option_name)
+        self.global_config["options"][option_name] = value
+        return old_val
```

Comment:
Why return the old value?

Reply:
Just some extra functionality for anyone using the method in the future

Reply:
Actually removing it, don't see why anyone would use this haha